nss-day-cohort-48/fun-website-1-digital-drops
# I added the birds.html

##To Test

1. Pull
2. git fetch --all
3. git checkout thejmdw-adding-birds
4. serve
5. I added HTML for the birds page. Check that there is a paragraph about birds, 3 bird photos, 3 bird titles, 7 facts.
